### PR TITLE
Days of week and months are localized properly now

### PR DESCRIPTION
**Pull request by achumakov**
Used date_i18n not WP_locale as either the latter behaves strange or I just didn't manage to understand that.

Anyway, with date_i18n both days and months work as they should.

### CODE REVIEW

File: modules/calendar/calendar.php [hunk 1]
```diff
@@ -690,10 +690,10 @@ function view_calendar() {
 				<?php if ( $split_month ): ?>
 				<tr class="month-marker">
 					<?php foreach( $week_dates as $key => $week_single_date ) {
-						if ( date( 'F', strtotime( $week_single_date ) ) != $split_month && date( 'F', strtotime( "+1 day", strtotime( $week_single_date ) ) ) == $split_month ) {
-							$previous_month = date( 'F', strtotime( $week_single_date ) );
+						if ( date_i18n( 'F', strtotime( $week_single_date ) ) != $split_month && date_i18n( 'F', strtotime( "+1 day", strtotime( $week_single_date ) ) ) == $split_month ) {
```
Comment (cojennin) [inline modules/calendar/calendar.php]: @achumakov Using `date_i18n` in these `if` statements won't cause any issues, right? Not very familiar with translations, so just wanted to double check.

**Comment by cojennin** [comment]
PR looks good! Just had a quick question about the the `date_i18n` usage, other than that everything seems to be working great!

Is there a way to test any of these functions to make sure they're localizing properly? Not familiar with PHPUnit and localizing, but might be good to add one in here!

**Comment by achumakov** [comment]
I've checked them in various locales, and analyzed the code. They work great _if_ all of them in the code block are either date_i18n or just date.

The code is just fetching month names of posts in the range, and if they differ, switches to a 'split month' output.

While all the checks are consistently calling the same function, all comparisons work with any locale.

**Comment by cojennin** [comment]
Gotcha. Alright, checked it out with a few different locales as well and it looks good, going to merge!

**Pull request merged**